tree-sitter-darklang/grammar.js

```diff
@@ -357,6 +357,14 @@ module.exports = grammar({
         $.pipe_expression,
 
         $.record_update,
+
+        $.qualified_const_name,
+      ),
+
+    qualified_const_name: $ =>
```

Comment:
Not sure if the naming is correct here, please let me know if it should be renamed